docs/templates/document-templates/plugin_README.md

```diff
@@ -0,0 +1,40 @@
+# {Name} {#required}
+
+> **NOTE**: Modify the `{Name}` and insert the name of your plugin. Use Heading 1 (H1).
+
+## Overview {#required}
+
+Provide a description of the plugin(s) functionality and on what types of resources is/are adapted.
```

Comment:
Provide a description of the plugin functionality. For example, describe the resources types on which it is adapted.